spec/features/user_add_items_to_carts_spec.rb

```diff
@@ -15,14 +20,28 @@
       expect(page).to have_css("#cart", text: "0")
       click_button "Add to cart"
       expect(page).to have_css("#cart", text: "1")
-    end
-
-    scenario "user chooses to add the same item again" do
-      visit foods_path
       3.times do
         click_button "Add to cart"
       end
-      expect(page).to have_css("#cart", text: "3")
+      expect(page).to have_css("#cart", text: "4")
+
+      visit carts_path
+      expect(page).to have_content("#{F1.name}")
+      expect(page).to have_content("#{F1.price}")
+      expect(page).to have_css(".line-total", text: "6000")
+
+      select('4', from: 'Select Box')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.